src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationProvider.cs

```diff
@@ -104,7 +84,7 @@ private void Load(bool reload)
         /// </summary>
         /// <exception cref="FileNotFoundException">If Optional is <c>false</c> on the source and a
         /// file does not exist at specified Path.</exception>
-        public override void Load()
+        public override void DoLoad()
```

Comment:
Curious, isn't the call to `Load(reload: true) that happens in the delegate we pass to `ChangeToken.OnChange()` also supposed to handle erros? I might be a bit confused already but from looking at the code, it seems to me that it doesn't because it doesn't go through the parameter-less `Load()`.